src/create-element.js

```diff
@@ -100,3 +79,35 @@ export function Fragment(props) {
  */
 export const isValidElement = vnode =>
 	vnode != null && vnode.constructor === undefined;
+
+export function createBackingNode(vnode, original) {
+	const backingNode = {
+		key: vnode.props ? vnode.props.key : undefined,
```

Comment:
Would using `null` here make it better for v8

Reply:
Yes.